std/string.d

```diff
@@ -4153,7 +4154,7 @@ unittest
         $(LREF entab)
   +/
 auto entabber(Range)(Range r, size_t tabSize = 8)
-    if (isForwardRange!Range)
+    if (isForwardRange!Range && isSomeChar!(ElementEncodingType!Range))
```

Comment:
I think I misunderstand.

what do you mean by "ranges that are not strings"? 

on line 4471 ranges are passed to entabber